ZZ_testing_channels.md

```diff
@@ -0,0 +1,163 @@
+Testing Phoenix Channels will help ensure that your code continues to work
+as you make changes to your application. It helps guide development and
+ensure there is no breakage later on down the road.
+
+Phoenix has built in helper functions in `Phoenix.Channel.Test` that help
+make testing Channels easy.
+
+### Testing `join` authorization
+
+Let’s add a new file at `test/channels/room_channel_test.exs`
+
+    defmodule App.RoomChannelTest do
```

Comment:
Up to now, we've used the application name of `HelloPhoenix` in all the guides. That may or may not be the best thing, but for consistency, let's continue to use it anywhere we use an application name to fully qualify a module name in this guide.